keps/sig-storage/1855-csi-driver-service-account-token/README.md

```diff
@@ -215,6 +220,51 @@ Option 1 is adopted. See discussion
 - **Are there any tests for feature enablement/disablement?** yes, unit tests
   will cover this.
 
+### Rollout, Upgrade and Rollback Planning
+
+- **How can a rollout fail? Can it impact already running workloads?**
+  Rollout will not fail because this change only exposes an extra field in CSIDriverSpec.
+
+* **What specific metrics should inform a rollback?**
+
+  - `storage_operation_duration_seconds`: if the corresponding csi plugin has
+    high error rates.
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+  No because this feature is disabled by default in 1.20.
```

Comment:
what will happen if the volume is created and then the apiserver is downgraded and the field disappears?

Reply:
the volume will still exist because this feature only passes extra volume attributes to in `NodePublishVolume` call. if the apiserver is downgraded, the volume attributes will no longer plumbed down.